packages/admin-e2e-tests/src/sections/onboarding/BusinessSection.ts

```diff
@@ -37,6 +37,27 @@ export class BusinessSection extends BasePage {
 
 		await sellingElsewhereDropdown.select( currentlySelling );
 	}
+	async selectEmployeesNumber( employeesNumber: string ) {
+		const employeesNumberDropdown = this.getDropdownField(
+			'.components-card__body > div:nth-child(3)'
```

Comment:
Instead of `div:nth-child(3)` are there more specific selectors we can use (or add) to make this test a little less brittle?

Reply:
Good idea! This was added in the commits c94c285 and 46d7568